compiler/dcalc/from_scopelang.ml

```diff
@@ -216,7 +220,7 @@ let input_var_typ typ io_in =
 let thunk_scope_arg var_ctx e =
   match var_ctx.scope_input_io, var_ctx.scope_input_thunked with
   | (Runtime.NoInput, _), _ -> invalid_arg "thunk_scope_arg"
-  | (Runtime.OnlyInput, _), false -> Expr.eerroronempty e (Mark.get e)
+  | (Runtime.OnlyInput, _), false -> e
```

Comment:
OK well spotted, indeed for true inputs of scopes we don't need error on empty because the input has been defined earlier and there's already an error on empty there. Is that the right reason?

Reply:
I think that's right, yes :)